iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -808,6 +808,14 @@ public override async Task EnableTwinPatchAsync(CancellationToken cancellationTo
             await _channel.WriteAsync(new SubscribePacket(0, new SubscriptionRequest(TwinPatchTopicFilter, QualityOfService.AtMostOnce))).ConfigureAwait(true);
         }
 
+        public override async Task DisableTwinPatchAsync(CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
```

Comment:
These would benefit from some log statements here, at Enter and Exit

Reply:
Totally, will add to my list of todos.